### PR TITLE
Fix datatypes page

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: src/admin/datatypes/index.md [hunk 1]
```diff
@@ -3,6 +3,5 @@
 Hub page for [administering](/src/admin/index.md) Galaxy data types.
 
 * [Admin/ReferenceMAFs](/src/admin/reference-mafs/index.md)
-* [Adding Completely new Datatypes](/src/admin/datatypes/Adding Complete Datatypes/index.md)
```
Comment (martenson) [inline src/admin/datatypes/index.md]: why was this dropped? it lives at https://galaxyproject.org/admin/datatypes/adding-complete-datatypes/

Comment (bgruening) [inline src/admin/datatypes/index.md]: it was not even in the old wiki: https://moin.galaxyproject.org/Admin/Datatypes and the link was broken. Will add it back.

Comment (bgruening) [inline src/admin/datatypes/index.md]: Ok, now?

File: src/admin/datatypes/composite-datatypes/index.md [hunk 2]
```diff
@@ -35,44 +35,44 @@ class BasicComposite(Text):
...
-
-```
+${os.path.join( input1.extra_files_path, 'results.dat' )} $output1</command>
+```
+
+If a tool is aware of the file names for a datatype, then only input1.extra\_files\_path needs to be provided.
+
+There are cases when it is desireable for the composite filenames to have varying names, but be of a similar form; for an example of this see Rgenetics below.
+
+Before continuing with this section, be sure to read and understand [AddingDatatypes](Admin%2FDatatypes%2FAdding+Datatypes).
```
Comment (dannon) [inline src/admin/datatypes/composite-datatypes/index.md]: This and any of the other links with %2F/etc in them probably need to be updated.

**Comment by dannon** [comment]
Looks good to me now!

File: src/admin/datatypes/index.md (hunk 1 above)
Comment (martenson) [inline src/admin/datatypes/index.md]: @bgruening that means hub is better than the wiki! :o)

**Pull request merged**